### ISSUE TITLE
This page type has no Block Areas configured

### ISSUE THREAD

**Issue by brasileric** [sheadawson/silverstripe-blocks#75]
Whatever I do in config.yml, I always get in the Blocks tab of the pages:

This page type has no Block Areas configured.

Tried (my theme is called main):

BlockManager:
  themes:
    main:
      BlockArea:
        Sidebar: true

BlockManager:
  themes:
    main:
      area:
        Sidebar: true

BlockManager:
  themes:
    main:
      area:
        ContentBlock: true

etc. etc.

SilverStripe 3.3.1

**Comment by Doniz** [comment]
did you add variables in your template as example below ?

`

    $BlockArea(BeforeContent)
    <% if $Content %><div class="row form">$Content</div><% end_if %>
    <% if $Form %><div class="row form">$Form</div><% end_if %>
    $BlockArea(AfterContent)

`

**Comment by sheadawson** [comment]
Yeap none of those will work. Read the documentation, there are very clear examples ;)

**Comment by brasileric** [comment]
Well, when I follow the documentattion exactly, the Blocks tab at the pages disappear and didn't set exclude_from_page_types for any page type.

**Comment by brasileric** [comment]
Update: when I use version 1.0.7. everything is working well. But with version 1.0.8. and dev my experience is that the Blocks tab disappears once you add an area to the page type. I experience this in SilverStripe 3.2.1 and 3.3.1.

**Comment by sheadawson** [comment]
Hmm... I've just done a test with 1.0.8 and was unable to replicate the issue. Can you paste in your config for me and I'll test with that

**Comment by witschko** [comment]
I can replicate this issue when I have not set the theme I use for blocks in my config.yml but only in the cms.

This has been set after a fresh installation of SilverStripe 3.3.1 without selecting the Simple theme:
```yaml
SSViewer:
  theme: 'tutorial'
```

I then installed the blocks module, created a new theme and selected it via the CMS. Then I followed the documentation and configured the blocks module (1.0.7). The blocks tab is shown but I get the message "This page type has no Block Areas configured". I then set the theme in config.yml to my new theme and it worked.

But when I install 1.0.8 or 1.0.9 the Blocks tab is not visible.
